docs/json/radarr/cf/71-surround.json

```diff
@@ -13,7 +13,7 @@
       }
     },
     {
-      "name": "5.1 Surround",
+      "name": "Not 5.1 Surround",
       "implementation": "ReleaseTitleSpecification",
       "negate": true,
       "required": true,
```

Comment:
We should delete the 5.1 negate in the 7.1 CF so that a misnamed file will always match the higher 7.1 CF